src/requirements/data/majors/bme.ts

```diff
@@ -57,84 +54,81 @@ const biomedicalEngineeringRequirements: readonly CollegeOrMajorRequirement[] =
       'Molecular, Cellular, and Systems Engineering (MCSE)': {
         description:
           'Must take CHEM 1570, BME 3110, BME 4190, and take 6 elective courses from the following courses.',
-        checker: includesWithSubRequirements(
-          ['CHEM 1570'],
-          ['BME 3020'],
-          ['BME 4190'],
-          ['BME 5830', 'BME 5850', 'BTRY 4381', 'BTRY 4840', 'CHEM 4810', 'CHEME 5430', 'CS 4780']
+        checker: includesWithSingleRequirement(
+          'CHEM 1570',
+          'BME 3020',
+          'BME 4190',
+          'BME 5830',
+          'BME 5850',
+          'BTRY 4381',
+          'BTRY 4840',
+          'CHEM 4810',
+          'CHEME 5430',
+          'CS 4780'
         ),
         counting: 'courses',
-        subRequirementProgress: 'any-can-count',
-        minCount: 13,
+        perSlotMinCount: [13],
```

Comment:
Same for this requirement as well as other reqs for the bme major. Why did we combine the `&` relation slots?